keypair/keypair.go

```diff
@@ -0,0 +1,124 @@
+package keypair
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/pkg/errors"
+)
+
+type Config struct {
+	PrivateKey *rsa.PrivateKey
+	PublicKey  *rsa.PublicKey
+	KeyPrefix  string
+	KeyPath    string
+}
+
+func (c *Config) GetPrivateKeyPath() string {
+	return fmt.Sprintf("%s/%s_private.pem", c.KeyPath, c.KeyPrefix)
+}
```

Comment:
What is the purpose for these rather than having the path to the key directly?

Reply:
No purpose. I think I just needed a ready way to save a file